roxie/ccd/ccdfile.cpp

```diff
@@ -379,9 +379,11 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
             }
             catch (IException *E)
             {
+                StringBuffer msg;
                 EXCLOG(MCoperatorError, E, "Read error");
+                E->errorMessage(msg);
                 E->Release();
-                OERRLOG("Failed to read length %d offset %" I64F "x file %s", len, pos, sources.item(activeIdx).queryFilename());
+                OERRLOG("Failed to read length %d offset %" I64F "x file %s [%s]", len, pos, sources.item(activeIdx).queryFilename(), msg.str);
```

Comment:
Won't the EXCLOG just above have already included this information?

Reply:
Does it? I thought the first parameter MCoperatorError was the target audience, and I thought OERRLOG also went to operator error audience.

Reply:
So it does.  Now I am confused why they claim it was not being reported....